pkgs/applications/version-management/radicle-cli/default.nix

```diff
@@ -2,61 +2,58 @@
 , stdenv
 , fetchFromGitHub
 , rustPlatform
-, pkg-config
-, cmake
 , installShellFiles
 , asciidoctor
 , DarwinTools
-, openssl
-, libusb1
 , AppKit
 , git
-, openssh
 , testers
 , radicle-cli
+, tree
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "radicle-cli";
-  version = "0.6.1";
+  version = "0.8.0-unstable-2023-07-26";
 
   src = fetchFromGitHub {
     owner = "radicle-dev";
-    repo = pname;
-    rev = "v${version}";
-    sha256 = "sha256-LS6zYpMg0LanRL2M8ioGG8Ys07TPT/3hP7geEGehwxg=";
+    repo = "heartwood";
+    rev = "7368e389427bc49e29e0554405c17f5c11c3c8de";
+    hash = "sha256-yhBYMxG4DkykEDeBbM2FZNq/Teb5R5A6kOp/Ei2mxEs=";
   };
 
-  cargoLock = {
-    lockFile = ./Cargo.lock;
-    outputHashes = {
-      "automerge-0.0.2" = "sha256-MZ1/rca8ZsEUhd3bhd502PHlBbvqAOtnWFEdp7XWmYE=";
-      "automerge-0.1.0" = "sha256-dwbmx3W13oZ1O0Uw3/D5Z0ht1BO1PmVVoWc/tLCm0/4=";
-      "cob-0.1.0" = "sha256-ewPJEx7OSr8X6e5QJ4dh2SbzZ2TDa8G4zBR5euBbABo=";
-      "libusb1-sys-0.6.2" = "sha256-577ld1xqJkHp2bqALNq5IuZivD8y+VO8vNy9Y+hfq6c=";
-      "walletconnect-0.1.0" = "sha256-fdgdhotTYBmWbR4r0OMplOwhYq1C7jkuOdhKASjH+Fs=";
-    };
-  };
+  cargoHash = "sha256-69qdH7qlXS0zK9lFPbl0IPmh01N/aIseGLBL1BNAtP0=";
 
-  # Otherwise, there are errors due to the `abigen` macro from `ethers`.
-  auditable = false;
+  patches = [
+    ./ignore-test-with-race-condition.patch
+    ./ignore-tests-requiring-terminal.patch
+  ];
```

Comment:
Thanks for your work on getting the tests to pass! It seems that we still have failures on `x86_64-darwin`: https://github.com/NixOS/nixpkgs/pull/244417/checks?check_run_id=15408058168

In general, I think it is acceptable to disable the tests (maybe just on non-`x86_64-linux`) if it turns out too difficult to get them to pass; especially as radicle is currently still on an experimental version.